content/articles/testing-angular-app-with-protractor-and-jasmine/index.md

```diff
@@ -0,0 +1,185 @@
+
+## Testing Angular applications using Protractor and Jasmine
+
+
+![protractor](./protractor.jpg "protractor")
```

Comment:
@Eze4Manuel, image paths take the following format :`/engineering-education/folder_name/image_name. Kindly make edits where applicable.